packages/automerge-doc-server/package.json

```diff
@@ -5,24 +5,30 @@
     "main": "dist/main.js",
     "scripts": {
         "main": "tsx src/main.ts",
+        "migrate-storage": "tsx src/populate_automerge_storage.ts",
```

Comment:
I assume we don't want to leave around this script forever. What is the intended lifecycle of this script? We run it first on staging and the eventually on production, then remove it?

Should I run it on staging immediately after merging this PR?